R/qenv-eval_code.R

```diff
@@ -79,6 +79,20 @@ setMethod("eval_code", signature = c("qenv", "character"), function(object, code
     }
 
     attr(current_code, "id") <- sample.int(.Machine$integer.max, size = 1)
+
+    # UNSURE if the removal of curly brackets is still needed.
+    tcode <- trimws(current_code)
+    if (any(grepl("^\\{.*\\}$", tcode))) {
+      tcode <- sub("^\\{(.*)\\}$", "\\1", tcode)
+    }
+
+    parsed_code <- parse(text = tcode, keep.source = TRUE)
```

Comment:
there is `parsed_code` in this loop, let's use the first one crated on the top

Reply:
t@gogonzo the first parsed_code is based on `code` and second one on `tcode`. But I can move `tcode` creation to top, and only use `tcode`. Let's see if this doesn't break anything